rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly"
+profile = "minimal"
+targets = [ "wasm32-unknown-unknown" ]
```

Comment:
Compare this to [Moonbeam's](https://github.com/moonbeam-foundation/moonbeam/blob/master/rust-toolchain)

```toml
[toolchain]
channel = "1.71.0"
components = [ "rustfmt", "clippy" ]
targets = [ "wasm32-unknown-unknown" ]
profile = "minimal"
```

A few questions:
1. Do we actually need nightly as the channel? If not, I prefer 1.72.0. I don't really know much about this though.
2. Should we list components like Moonbeams?

Reply:
I need to study the toolchain file and set the right parameters, I was hoping you were an expert in that matter. 
The current one reflects the settings I found in the old CI script